src/test/script_standard_tests.cpp

```diff
@@ -490,8 +490,8 @@ BOOST_AUTO_TEST_CASE(script_standard_IsMine)
     {
         CBasicKeyStore keystore;
 
-        CScript redeemscript = GetScriptForDestination(pubkeys[0].GetID());
-        CScript witnessscript = GetScriptForDestination(CScriptID(redeemscript));
+        CScript redeemscript = GetScriptForDestination(ScriptHash(PKHash(pubkeys[0])));
```

Comment:
Why is there `ScriptHash` here?

Reply:
hmm seems I broke this test case, thanks